packages/pluggableWidgets/bar-chart-native/src/BarChart.tsx

```diff
@@ -0,0 +1,33 @@
+import { createElement, ReactElement } from "react";
+import { all } from "deepmerge";
+
+import { BarChartProps } from "../typings/BarChartProps";
+import { BarChart as BarChartComponent } from "./components/BarChart";
+import { BarChartStyle, defaultBarChartStyle } from "./ui/Styles";
+import { useSeries } from "./utils/SeriesLoader";
+
+export function BarChart(props: BarChartProps<BarChartStyle>): ReactElement | null {
+    const { name, barSeries, style, xAxisLabel, yAxisLabel, showLabels, ...rest } = props;
+
+    const customStyles = style ? style.filter(o => o != null) : [];
```

Comment:
`style.filter(o => o != null) ?? []`

Reply:
`style?.filter(o => o != null) ?? []` may work 👍

Reply:
Ah, `style` is always an array, so indeed your suggestion works fine. Thanks.

Reply:
👍